posix.mak

```diff
@@ -204,7 +204,7 @@ $(IMPDIR)/%.d : src/%.d
 ######################## Build DMD if non-existent ##############################
 
 $(DMD):
-	make -C $(DMD_DIR)/src -f posix.mak BUILD=$(BUILD) OS=$(OS) MODEL=$(MODEL)
+	$(MAKE) -C $(DMD_DIR)/src -f posix.mak BUILD=$(BUILD) OS=$(OS) MODEL=$(MODEL)
```

Comment:
I see one more on line 541, don't know if you care about that one.

Reply:
I just moved the posix.mak file to the other PR (#2002), because it was mostly needed to compile the dragonflybsd specific files, so a better fit (and i hope it will fix the autotester issues when compiling on win32).
I did checkout the posix.mak file,  but it only has 374 lines.

Reply:
Whoops, checked the [Phobos posix.mak for `make` by mistake](https://github.com/dlang/phobos/blob/master/posix.mak#L531).

Reply:
Moving the make files to the other PR, seems to have been a good idea. It fixed the autotester win32 issue.
I will check the phobos posix.mak file, to see if it needs correcting (And create a PR if necessary).